src/sentry/integrations/slack/actions/notification.py

```diff
@@ -86,45 +83,25 @@ def send_notification(event: GroupEvent, futures: Sequence[RuleFuture]) -> None:
             additional_attachment = get_additional_attachment(
                 integration, self.project.organization
             )
-            payload = {}
-            if features.has("organizations:slack-block-kit", event.group.project.organization):
-                blocks = SlackIssuesMessageBuilder(
-                    group=event.group,
-                    event=event,
-                    tags=tags,
-                    rules=rules,
-                    notes=self.get_option("notes", ""),
-                ).build(notification_uuid=notification_uuid)
-
-                if additional_attachment:
-                    for block in additional_attachment:
-                        blocks["blocks"].append(block)
+            blocks = SlackIssuesMessageBuilder(
+                group=event.group,
+                event=event,
+                tags=tags,
+                rules=rules,
+                notes=self.get_option("notes", ""),
+            ).build(notification_uuid=notification_uuid)
 
-                if blocks.get("blocks"):
-                    payload = {
-                        "text": blocks.get("text"),
-                        "blocks": json.dumps(blocks.get("blocks")),
-                        "channel": channel,
-                        "unfurl_links": False,
-                        "unfurl_media": False,
-                    }
-            else:
-                attachments = [
-                    SlackIssuesMessageBuilder(
-                        group=event.group,
-                        event=event,
-                        tags=tags,
-                        rules=rules,
-                    ).build(notification_uuid=notification_uuid)
-                ]
-                # getsentry might add a billing related attachment
-                if additional_attachment:
-                    attachments.append(additional_attachment)
+            if additional_attachment:
+                for block in additional_attachment:
+                    blocks["blocks"].append(block)
 
+            if blocks.get("blocks"):
```

Comment:
nit: you could do walrus command here so that line 101 is a bit easier
```suggestion
            if payload_blocks := blocks.get("blocks"):
                   payload = {json.dumps(payload_blocks)}
```

Reply:
i forget this is a thing, thanks!